test/with_logabsdet_jacobian.jl

```diff
@@ -0,0 +1,20 @@
+@testset "with_logabsdet_jacobian.jl" begin
+    derivative(f, x) = ChainRulesTestUtils.frule((ChainRulesTestUtils.NoTangent(), 1), f, x)[2]
+
+    x = randexp()
+
+    ChangesOfVariables.test_with_logabsdet_jacobian(log1pexp, x, derivative)
+    ChangesOfVariables.test_with_logabsdet_jacobian(log1pexp, -x, derivative)
+
+    ChangesOfVariables.test_with_logabsdet_jacobian(logexpm1, x, derivative)
+
+    ChangesOfVariables.test_with_logabsdet_jacobian(log1mexp, -x, derivative)
+
+    ChangesOfVariables.test_with_logabsdet_jacobian(log2mexp, log(2) - x, derivative)
+    ChangesOfVariables.test_with_logabsdet_jacobian(log2mexp, x - log(2), derivative)
```

Comment:
The input to `log2mexp` has to be <= log(2). I guess the line above should be sufficient:
```suggestion
```

Reply:
Yes, just took that out.